spec/appendix.md

```diff
@@ -1,12 +1,57 @@
+* not all ToIP definitions are fully linked here. This whole section is more of an exemplar as of the time of writing.
+
+[[def: autonomous identifier]] 
+~ Another term for [[ref: self-certifying identifier (SCID)]]. 
+
+[[def: cryptographic keys]]
+~ A key in cryptography is a piece of information, usually a string of numbers or letters that are stored in a file, which, when processed through a cryptographic algorithm, can encode or decode cryptographic data. Symmetric cryptography refers to the practice of the same key being used for both encryption and decryption. Asymmetric cryptography has separate keys for encrypting and decrypting. These keys are known as the public keys and private keys, respectively. Source: Wikipedia.
+
+[[def: cryptographically verifiable]]
+~ A property of a data structure that has been digitally signed using a private key such that the digital signature can be verified using the public key. Verifiable data, verifiable messages, verifiable credentials, and verifiable data registries are all cryptographically verifiable. Cryptographic verifiability is a primary goal of the ToIP Technology Stack.
+
+[[def: identifier]]
+~ A single attribute—typically a character string—that uniquely identifies an entity within a specific context (which may be a global context). Examples include the name of a party, the URL of an organization, or a serial number for a man-made thing. Supporting definitions: eSSIF-Lab: a character string that is being used for the identification of some entity (yet may refer to 0, 1, or more entities, depending on the context within which it is being used).
+
+[[def: self-certifying identifier (SCID), SCID, SCIDs]]
+~ A subclass of verifiable identifier ([[ref: VID]]) that is [[def: cryptographically verifiable]] without the need to rely on any [[ref: third party]] for [[ref: verification]] because the [[ref: identifier]] is cryptographically bound to the [[ref: cryptographic keys]] from which it was generated. Also known as: [[ref: autonomous identifier]].
+
+[[def: third party]]
+~ A party that is not directly involved in the trust relationship between a first party and a second party, but provides supporting services to either or both of them.
+
+[[def: verifiable identifier (VID), VID, VIDs]]
+~ An identifier over which the controller can provide cryptographic proof of control.
+See also: decentralized identifier, [ref: self-certifying identifier (SCID)].
+
+[[def: verification]]
+~ An action an agent (of a principal) performs to determine the authenticity of a claim or other digital object using a cryptographic key.
+
+#### Glossary WORK TO BE DONE
+
+TODO: 
+
+* DECISION to duplicate ToIP Glossary terms here (for completeness of this spec) or include them only by reference.
```

Comment:
I lean towards linking to the ToIP glossary for the longer list of more general terms, like identifier.